stl/inc/ostream

```diff
@@ -109,7 +109,7 @@ public:
         _STL_DISABLE_DEPRECATED_WARNING
         __CLR_OR_THIS_CALL ~sentry() noexcept {
 #if _HAS_EXCEPTIONS
-            if (!_STD uncaught_exception()) { // TRANSITION, OS#15518458
+            if (!_STD uncaught_exception()) { // TRANSITION, was OS issue 15518458

```

Comment:
This is why I prefer to always have the intended code side-by-side with the actual code in a preprocessor conditional when annotating `TRANSITION`s: this bug is gone, so we now have "no way" to determine what it's trying to tell us (No change requested, unless you happen to know why this is commented in which case it would be nice to replace the comment.)

Reply:
(Apologies to externals: I think most of these links are inaccessible to you.)

Aha! I figured out where the old OS bugs go: they [are archived](https://osgwiki.com/wiki/VSTS_Archiving) under [a distinct Azure DevOps project](https://microsoftarchive.visualstudio.com/) to keep the OS repo from accumulating so many work items as to be unusable. 

This bug is now https://microsoftarchive.visualstudio.com/OS/_workitems/edit/12000909, for example. Shall we add an `OS-archive-<meow>` autolink?

Reply:
Sure. I've chosen `ArchivedOS-` as a prefix. Thanks for finding this!